modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -328,28 +370,17 @@ public void run() {
          * @param lease Old lease to apply CAS in Meta storage.
          * @param candidate Lease candidate.
          */
-        private void writeNewLeaseInMetaStorage(ReplicationGroupId grpId, Lease lease, ClusterNode candidate) {
-            var leaseKey = ByteArray.fromString(PLACEMENTDRIVER_PREFIX + grpId);
-
+        private void writeNewLeaseInMetaStorage(ReplicationGroupId grpId, Lease lease, ClusterNode candidate,
```

Comment:
The method is not writing something to MS anymore. I think, it should be renamed with correction of java doc.

Reply:
fixed